### ISSUE TITLE
implement staged claim

### ISSUE THREAD

**Issue by LaurentMontBlanc** [planetmint/planetmint-go#184]
## Description
Based on a PoP result that is sent to Planetmint a certain amount of staged claims is minted to an account. Once a reissuance is triggered staged claims are converted to claims. We need to implement the mechanisms to mint, burn and convert between 

## Tasks
- [x] implement staged claim mint
- [x] calculate staged claim
- [x] implement staged claim burn & claim mint

### Implement staged claim mint/burn & claim mint/burn
On every PoP result mint staged claim token to the challenger/challengee. On every re-issuance iterate over all challenges between blockheight of re-issuance at height n and n-1.

**Comment by LaurentMontBlanc** [comment]
@eckelj I've tried to test this locally but got stuck on the `err = tx.GenerateOrBroadcastTxWithFactory(clientCtx, txf, msgs...)` call in `lib/tx.go:154`, because the application froze. 
I've tried setting the `ChainID` and `BroadcastMode` in the `getClientContextAndTxFactory(goCtx context.Context, address sdk.AccAddress) (clientCtx client.Context, txf tx.Factory, err error)` at `lib/tx.go:65`.
I cannot test this isolated on the `dao/keeper/keeper.go::Keeper` as this relies on the BankKeeper. 

The following local test scenario could work to validate this manually:
- remove the BeginBlocker in `dao/module.go::AppModule:BeginBlock` (this automatically sends PoPs, Reissuances and Distributions)
- send some txs to `ReportPoPResult` (this creates staged claims)
- check balances of challenger and challengee address for `stagedcrddl`
- send a tx to `DistributionRequest` from a different address than the validator (this stores some DistributionOrders without liquid interaction)
- send a tx to `DistributionResult` (this resolves the staged claims to claims)
- check balances of challanger and challangee for `crddl`
